htlcswitch/switch_test.go

```diff
@@ -3377,6 +3377,65 @@ func TestSwitchHoldForward(t *testing.T) {
 	assertNumCircuits(t, s, 0, 0)
 
 	require.NoError(t, switchForwardInterceptor.Stop())
+
```

Comment:
Add a case for clearing out of forwarded packets when we're not running in always-on mode and connect/disconnect/reconnect?

Reply:
Added. This was pre-existing though.